content/engine/security/rootless.md

```diff
@@ -70,21 +72,22 @@ testuser:231072:65536
   script](https://get.docker.com/rootless), however, you must add an AppArmor
   profile for `rootlesskit` manually:
 
-  1. Add the AppArmor profile to `/etc/apparmor.d/usr.local.bin.rootlesskit`:
+  1. Create and install the currently logged-in user's AppArmor profile:
 
      ```console
-     $ cat <<EOF > /etc/apparmor.d/$(echo $HOME/bin/rootlesskit | sed -e s@^/@@ -e s@/@.@g)
```

Comment:
Maybe just instruct to run the old commands as root

Reply:
Not all users have `sudo` access and the old command includes  `$HOME` which will always be `/root` when run as root. The old command appears to have never been and is not as versatile as the new.